docs/cedarling/patterns/annotation-workflow.md

```
@@ -0,0 +1,426 @@
+A merchant who has submitted business details but nothing else gets both remaining steps back in
+one call:
+
+```text
+DENY   reason: forbid_accept_payments_without_bank_account
+               forbid_accept_payments_without_identity_check
+```
+
+Four annotations describe each missing step, and they only mean anything together, so read them
+with `annotations_by_policy` and keep each policy's set intact:
+
+```rust
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)